utils.js

```diff
@@ -0,0 +1,22 @@
+export const isElementAlreadyRegistered = (elName) => {
+  var registeredElement;
+  if (window.Polymer && Polymer.telemetry) {
+    registeredElement = Polymer.telemetry.registrations.filter((el) => el.is == elName);
+
+    if (registeredElement) {
+      console.warn(elName, "is already registered as a custom polymer element.");
```

Comment:
It's just utility function, so `warn` log shouldn't be here. It's user should put `warn` log as needed basis.